controllers/teams.js

```diff
@@ -634,6 +635,12 @@ router.get('/:teamId', async (req, res, next) => {
 
 		const permissions = await api(req).get(`/teams/${teamId}/userPermissions/${course.user.userId}`);
 
+		const nextcloudUrl = Configuration.get('NEXTCLOUD_REDIRECT_URL') !== ''
```

Comment:
do we need a null check first?

Reply:
no, we dont a null check. the default schema sets a default value for it